stdlib/obj.ml

```diff
@@ -18,8 +18,9 @@ type t
 external repr : 'a -> t = "%identity"
 external obj : t -> 'a = "%identity"
 external magic : 'a -> 'b = "%identity"
-external is_block : t -> bool = "caml_obj_is_block"
 external is_int : t -> bool = "%obj_is_int"
+let is_block a = is_int a |> not
+  [@@ocaml.inline]
```

Comment:
Indeed, `not (is_int a)` is both easier to read and easier to compile, no need to add a function call here.

As for `[@@inline]` vs `[@@ocaml.inline]`, we should standardize on something. Currently in the standard library all `deprecated` attributes are `ocaml.deprecated`, while `unboxed` and `noalloc` are unqualified.

Personally I think it's better to use the qualified attribute, and to specify `always` explicitly even if it's the default.